test/acceptance_tests/conftest.py

```diff
@@ -0,0 +1,96 @@
+from uuid import uuid4
```

Comment:
License Headers

Reply:
I thought we didn't need the license in `./test/...`